fpr/models.py

```diff
@@ -76,6 +76,10 @@ class Meta:
         verbose_name = _("Format")
         ordering = ['group', 'description']
 
+    def get_pronom_ids(self):
+        s = FormatVersion.objects.filter(format=self).values_list('pronom_id',flat=True)
+        return ", ".join(s)
```

Comment:
With this new function being called on each item in the list, the view went from making 1031 db queries (ridiculous amount) to 2061 db queries and the total response time took twice as much. This is a problem we already have in other list views in this application, e.g. Identification > Rules makes more than 4k db queries.

I think that we could do much better that this, but it doesn't need to happen now. A simple solution could be to introduce [pagination](https://docs.djangoproject.com/en/2.1/topics/pagination/), not sure why we're not doing this already! Pagination wouldn't solve our inefficient queries to the database but it would limit its scope significantly which is probably good enough given our limited resources. I'll file an issue.

Reply:
https://github.com/archivematica/Issues/issues/138

Reply:
Yeah I noticed we paginate after the fact, just visually and not optimizing the queries at all. 

I thought about this db-querying and noticed the page-load wasn't that much worse... As we've discussed before, I agree with you that maybe it's better to not try to optimize something right now that needs a much larger change overall.